app/src/main/res/layout/top_site_item.xml

```diff
@@ -41,6 +41,26 @@
         app:layout_constraintLeft_toLeftOf="parent"
         app:layout_constraintRight_toRightOf="parent"
         app:layout_constraintTop_toBottomOf="@id/favicon_card"
-        tools:ignore="RtlCompat,SmallSp" />
+        tools:ignore="RtlCompat,SmallSp"
+        tools:text="Mozilla"/>
+
+    <TextView
+        android:id="@+id/top_site_subtitle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:maxWidth="84dp"
```

Comment:
Is the max width a precaution so this doesn't grow beyond the width of a top site?

Reply:
I believe that is the intent.